projects/bp-gallery/src/components/views/search/SearchBreadcrumbs.scss

```diff
@@ -20,7 +20,7 @@
     &[search-type='description'] {
       background: $descriptionColor;
     }
-    &[search-type='decade'] {
```

Comment:
Shouldn't we maybe keep 'decade' as well? When clicking on an Item in the DecadesList, there is still a breadcrumb (with search-type: 'decade')

Reply:
That is sth. we should keep yeah 👍